src/App.jsx

```diff
@@ -1,10 +1,58 @@
-import { DemonstratingProps } from "./components/demo/DemonstratingProps";
+// import { DemonstratingProps } from "./components/demo/DemonstratingProps";
+import React from "react";
+import axios from "axios";
+import MovieCard from "./MovieCard";
+
+// @ts-ignore - supress an error about import.meta
+const baseURL = import.meta.env.VITE_API_BASE_URL;
 
 function App() {
+    const [searchTerm, setSearchTerm] = React.useState("");
+    const [movieList, setMovieList] = React.useState([]);
+    const [movieToDisplay, setMovieToDisplay] = React.useState({});
+
+    function handleSearchInput(event) {
+        setSearchTerm(event.target.value);
+    }
+
+    const handleSearch = async () => {
```

Comment:
variable handleSearch can be described clearer e.g handleMovieSearch. in case there are more search functions